cmd/shield/main.go

```diff
@@ -1188,6 +1192,36 @@ func main() {
 		r.Add("Name", t.Name)
 		r.Output(os.Stdout)
 
+	/* }}} */
+	case "delete-tenant": /* {{{ */
+		if len(args) != 1 {
+			fail(2, "Usage: shield %s [OPTIONS] NAME-or-UUID\n", command)
+		}
+		t, err := c.FindTenant(args[0], true)
+		bail(err)
+
+		if !confirm(opts.Yes, "Are you sure you want to delete all configuration under this tennant?") {
+			break
+		}
+
+		if !opts.DeleteTenant.Recursive {
+			_, err = c.DeleteTenant(t, false)
+			bail(err)
+		} else {
+			_, err = c.DeleteTenant(t, true)
+			bail(err)
+		}
+
+		if opts.JSON {
```

Comment:
I don't know how the other delete commands work, so I could be wrong here. Just seems counterintuitive to print out the thing you're deleting after its been confirmed that you're deleting it. Probably can skip printing out the actual tenant and just something along the lines of "YEET, THIS BEECH EMPTY"